README.md

```diff
@@ -71,14 +71,15 @@ This is available in both Schedule and Room views
 examples: 
 * `http://signs.scale.lan/?year=2019&month=3&day=7&hour=9&minute=10`
 * `http://signs.scale.lan/?room=ballroom-de&year=2019&month=3&day=8&hour=14&minute=33`
+> **NOTE:** This will not work unless `$starttime` is correctly set for the current year of the show
 
 ### Yearly Tasks
 
 There is a bit of manual effort necessary from year to year. These tasks include, but might not be limitted to:
-* update the logo for the curent year at `/server/images/header.png`
+* update the logo for the curent year at `/server/images/header.png` should be `360x170px`
```

Comment:
new one is 381x170. probably close enough given transparency

Reply:
I noticed this too. I believe it was due to: `convert header.png -resize 360x170^ new.png` where `^` allows for some play given the original file since I didnt want to distort it.

It looks good regardless :)